packages/payments-plugin/src/mollie/mollie.helpers.ts

```diff
@@ -34,7 +34,7 @@ export function toMollieOrderLines(order: Order, alreadyPaid: number): CreatePar
         quantity: line.quantity,
         unitPrice: toAmount(line.proratedLinePriceWithTax / line.quantity, order.currencyCode), // totalAmount has to match unitPrice * quantity
         totalAmount: toAmount(line.proratedLinePriceWithTax, order.currencyCode),
-        vatRate: String(line.taxRate),
+        vatRate: line.taxRate.toFixed(2),
```

Comment:
Send as `20.00` to Mollie instead of `20`, otherwise comparison of new order vs existing order fails, because vat rate's dont equal.